legend-engine-ide-lsp-server/src/main/java/org/finos/legend/engine/ide/lsp/server/LegendLanguageServer.java

```diff
@@ -1057,4 +1068,23 @@ public static void main(String[] args)
         launcher.startListening();
         LOGGER.debug("Server launched");
     }
+
+    private static Properties loadVersions()
+    {
+        Properties properties = new Properties();
+
+        try (InputStream stream = LegendLanguageServer.class.getClassLoader().getResourceAsStream("versions.properties"))
+        {
+            if (stream != null)
+            {
+                properties.load(stream);
+                return properties;
+            }
+        }
+        catch (Exception e)
+        {
+            e.printStackTrace();
```

Comment:
We should either log this or throw

Reply:
Added to log.  I dont want the process to fail as this is for information purposes so it easy to track potential exceptions from incompatible versions (we built with engine 4.40, but user is trying 4.50 and that could not be backward compatible)